lib/rspec/rails/matchers/active_job.rb

```diff
@@ -11,7 +11,9 @@ module ActiveJob
         # rubocop: disable Style/ClassLength
         # @private
         class Base < RSpec::Matchers::BuiltIn::BaseMatcher
-          def initialize
+          def initialize(verb_present_tense, verb_past_tense)
+            @verb_present_tense = verb_present_tense
+            @verb_past_tense = verb_past_tense
```

Comment:
How do you feel about making these constants and defined appropriately as frozen strings in the sub matchers?

Reply:
I see three viable options:

1. keep as is (only add `.freeze`)

2. define methods with `freeze`
```ruby
def x
  'qwrqwfqw'.freeze
end
x.equal? x # => true
x.object_id == x.object_id
```
Seems that some Ruby magic is going on, and it's being optimized to reduce allocations.
The downside is that we have two method definitions for no good reason, and that doubles the number of methods in those matchers.

3. define class-level `attr_reader`, and use `self.class.verb_present_tense`

Do you have a preference here?

Reply:
Speaking about the names, I'm out of good ideas.
`message_expectation_action`/`failure_message_expectation_action`?